src/main/java/JavaQuestionTwo/home.java

```diff
@@ -0,0 +1,84 @@
+package JavaQuestionTwo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectOutputStream;
+import java.util.*;
+public class home {
+	Scanner sc = new Scanner(System.in);
+	public static void main(String[] args) {
+
+		home homeObj = new home();
+		homeObj.createFile();
+		homeObj.takeUserInput();
+
+	}
+
+	private void createFile() {
+		String filename = "userdata.txt";
```

Comment:
make this function generic, such that it takes a file name as input argument, instead of hardcoding file name inside.